src/clj/rems/migrations/migrate_organizations.clj

```diff
@@ -0,0 +1,71 @@
+(ns rems.migrations.multiple-forms
```

Comment:
wrong ns name

... actually, I think this whole file is unnecessary

Reply:
Yeah this was accidentally included because I started work on the migrations. I'll complete this one.

Reply:
sooo remove this file? it looks like the other one is finished now